src/components/Header/Header.styles.js

```diff
@@ -0,0 +1,59 @@
+import styled from 'styled-components';
+
+const Container = styled.div``;
+
+const Search = styled.input``;
+
+const RightHolder = styled.span`
+  float: right;
+`;
+
+const BurgerButton = styled.input`
+  width: 30px;
+  height: 30px;
+`;
+
+const ProfileButton = styled.input`
+  width: 50px;
+  height: 30px;
+`;
+
+const ToggleButton = styled.input`
+  -webkit-appearance: none;
+  position: relative;
+  outline: none;
+  width: 50px;
+  height: 25px;
+  background-color: blue;
+  border-radius: 12px;
+
+  &:before {
+    content: '';
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 25px;
+    height: 25px;
+    background-color: lightgray;
+    border-radius: 12px;
+  }
+
+  &:checked {
+    background-color: blue;
+  }
+
+  &:checked:before {
+    transform: translate(100%);
+  }
+`;
+
+const Styled = {
+  Container,
+  Search,
+  RightHolder,
+  BurgerButton,
+  ProfileButton,
+  ToggleButton,
+};
+
+export default Styled;
```

Comment:
Good use of `styled-components`, an alternative to the way you are exporting the components is something like:

```jsx
export const Container = styled.div``;

export const Search = styled.input``;

export const RightHolder = styled.span`
  ....
`;

... rest of components ...
```

this way you can delete from line 50 to 59 because JS will build the object for you, using the `export` before the variable definition will add the value of that variable to the object that will be built and when you need to use a component defined here you can import them like:

```jsx
import { Container, Search, RightHolder, ... } from './Styled';
```

by doing so you won't have to write `Styled.` everytime you want to use a component